fe/fe-core/src/main/java/org/apache/doris/analysis/CancelCloudWarmUpStmt.java

```diff
@@ -0,0 +1,85 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.analysis.BinaryPredicate.Operator;
+import org.apache.doris.common.AnalysisException;
+
+public class CancelCloudWarmUpStmt extends CancelStmt {
+    private Expr whereClause;
+    private long jobId;
+
+    public CancelCloudWarmUpStmt(Expr whereClause) {
+        this.whereClause = whereClause;
+    }
+
+    public long getJobId() {
+        return jobId;
+    }
+
+    @Override
+    public void analyze(Analyzer analyzer) throws AnalysisException {
```

Comment:
throw exception if not cloud mode

Reply:
这些文件 不要放到这个包目录下？
org/apache/doris/cloud/analysis 可以么？
```suggestion
    public void analyze(Analyzer analyzer) throws AnalysisException {
```